polyfill/test/usercalendar.mjs

```diff
@@ -293,6 +293,9 @@ describe('Userland calendar', () => {
       },
       fields(fields) {
         return fields.slice();
```

Comment:
Unrelated to this PR, but why does this implementation copy the object if it's not needed to be copied?

Reply:
It's specified that Calendar.fields() should make a copy. There's probably no need to actually do it, but this is just a throwaway calendar anyway.